R/utils.R

```diff
@@ -47,18 +47,21 @@ pmean <- function(..., na.rm = FALSE) {
 #' Transform an intervals set to matrix
 #'
 #' @param df intervals set
+#' @param remove_intervalID whether to remove the "intervalID" column generated by a gextract call
 #'
 #' @return matrix with coordinate rownames (chrom_start_end).
```

Comment:
Let's add also at the return value docs that if remove_intervalID is TRUE - it would be removed by default